Python/pywarpx/picmi.py

```diff
@@ -1936,6 +1976,50 @@ def solver_initialize_inputs(self):
                 self.Jz_external_function, self.mangle_dict
             ),
         )
+        pywarpx.hybridpicmodel.add_external_fields = self.add_external_fields
+        if self.add_external_fields:
```

Comment:
It might be a little cleaner to remove `self.add_external_fields` in the constructor and here do:
```suggestion
        if self.A_external is not None:
                pywarpx.hybridpicmodel.add_external_fields = True
```
It all should work fine then as long as you use `query` and not `get` for the runtime parameter `add_external_fields` with the parser.